lnrpc/rpc.proto

```diff
@@ -2096,8 +2096,7 @@ message PendingChannelsResponse {
     /// Channels pending opening
     repeated PendingOpenChannel pending_open_channels = 2;
 
-    /// Channels pending closing
-    repeated ClosedChannel pending_closing_channels = 3;
+    reserved 3;
```

Comment:
Not sure if we should first set it to `pending_closing_channels = 3 [deprecated = true];` first and just not fill it any more? Then remove it in the next version? Not sure what the `reserved` does for clients that use a previous RPC client?

Reply:
Agree, I too think deprecating first is the better way to do this. (Can even leave it deprecated).